[Cleanup] Confirmation Modals | Document Actions

Civolilah opened the pull request:
@beganovich @turbo124 This PR includes the implementation of confirmation modals for "Clone", "Void", and "Clone to template" document actions. Please let me know if there are any other actions that need confirmation modals.

turbo124 commented:
@Civolilah 

looks good, but feels like we are missing supplementary text for the action itself.

<img width="1353" height="725" alt="image" src="https://github.com/user-attachments/assets/ca560731-b8a9-409b-8129-789a1a8e83bb" />

Civolilah commented:
> @Civolilah
> 
> looks good, but feels like we are missing supplementary text for the action itself.
> 
> <img alt="image" width="1353" height="725" src="https://private-user-images.githubusercontent.com/5827962/509202504-ca560731-b8a9-409b-8129-789a1a8e83bb.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjIyODAxNzksIm5iZiI6MTc2MjI3OTg3OSwicGF0aCI6Ii81ODI3OTYyLzUwOTIwMjUwNC1jYTU2MDczMS1iOGE5LTQwOWItODEyOS03ODlhMWE4ZTgzYmIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MTEwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTExMDRUMTgxMTE5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YTQwNmU5N2RiMWExYTVlY2M5Njk3MjIyYzg5YThlZGUwYzc0YjQwN2Q3MjEyMjBjY2U5MzU4NjllZDcyNThjNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.sJZnvO-YzhYDUOZtfYE6XOIk2MznRKqQieXxZgSCiew">

@turbo124 Sure thing, I've added a confirmation message for each modal. Screenshot:

<img width="539" height="294" alt="Screenshot 2025-11-04 at 19 11 51" src="https://github.com/user-attachments/assets/3d294ef9-c09f-404d-ad57-e35ed8fb9004" />

So, I've added "clone_document_description", "make_template_description", and "void_document_description" translation keywords, so if you agree, please just add those translation keywords to the files.

Let me know your thoughts.
